cl_dll/hud_msg.cpp

```diff
@@ -69,8 +73,16 @@ void CHud :: MsgFunc_ViewMode( const char *pszName, int iSize, void *pbuf )
 
 void CHud :: MsgFunc_InitHUD( const char *pszName, int iSize, void *pbuf )
 {
+	BEGIN_READ(pbuf, iSize);
+
+	// Here we'll get a -1 if the server is AG 6.6 or less, because it's not sent at all, and 1 for AG 6.7
+	const auto slopefix = (READ_BYTE() == 1) ? 1.0f : 0.0f;
+	update_slopebug_fix(slopefix);
+
+	gEngfuncs.Con_DPrintf("Slopefix is %sactive on this server.\n", slopefix ? "" : "not ");
+
 	// prepare all hud data
-	HUDLIST *pList = m_pHudList;
+	HUDLIST* pList = m_pHudList;
```

Comment:
This is some automatic formatting performed by Visual Studio, I'll look into disabling it since I have had to undo manually lots of these before and I didn't realize of this one when checking with `git diff`.